src/multiplayer-servers/getting-started/cloud-provider-setup.md

```diff
@@ -53,3 +53,294 @@ In order to link your billing account to the projects managed by Nitrado, and to
 #### Confirming the setup
 
 After you have completed the steps above, please contact Nitrado to confirm that the setup is complete.
+
+## Amazon Web Services
+
+### Prerequisites
+
+* You **must** already have an existing AWS account
+* You **must** possess the necessary permissions to create IAM resources in your account
+
+### Background
+
+For AWS, to access another organization, you have to setup a chain of _roles_. Roles are AWS objects that allow scoping permissions. Roles are assumable by _principals_. Principals are arbitrary entities operating AWS, and specifically, users or groups of users. This setup allows you to delegate game server cluster management to Gamefabric operators by creating a role that is fine-grained to provision resources _without having to grant broad access to their organization_.
+
+By chaining those roles together (i.e., allowing a role to assume another role), Gamefabric operators can access your organization/account,
+as long as the chain of _Trust Relationship_ is not broken.
+
+<img style="padding: 2rem; box-sizing: border-box" src="./images/cloud/aws-iam-setup.png"/>
+
+For setting this up, we recommend using a tool like Terraform, as it allows you to simply declare the desired resources. Particularly for configuring various policy documents, this will be helpful. We'll use Terraform in the following. You can adapt this guide and do the following steps manually.
+
+### Creating the Role and Assumption Policy
+
+Create an IAM Role and the appropriate role assumption policy using Terraform:
+
+```terraform
+variable "source_saml_principal_arn" {
+  type        = string
+  description = "ARN of the principal backing the SAML authentication for SSO users from the source account"
+  default     = "arn:aws:iam::339712714940:saml-provider/AWSSSO_5495e4acbcad9a8a_DO_NOT_DELETE"
```

Comment:
Is it okay to have this in here?

Reply:
Well, you need this for SSO to work on our end. This ARN is also statically generated by AWS on creation of the account, and the Org ID is needed as well at some other places. You can't actually do anything with that object's ARN except for granting permissions to the principal it refers to, so in terms of sensitivity, I think this is okay to be left here, otherwise we can also only hand that out on request, but that then involves more human interaction again and this guide should be fairly independent from us for the most part.